ja-jp/Content/Topics/DeploymentGuides/Sharepoint/Guide/about-people-picker.htm

```diff
@@ -369,7 +348,7 @@ <h2>重要事項</h2>
                                     <!-- *********************** Footer rows ********************************* -->
                                     <div class="footer2" data-test="footer">
                                         <div>
-                                            <p class="copyright" data-test="copyrightNotice">&copy;  <span class="mc-variable okta-variables.Year variable">2022</span> <span class="mc-variable okta-variables.CompanyName variable">Okta, Inc</span>. 全著作権所有。それぞれの商標は、それぞれの商標所有者に帰属します。
+                                            <p class="copyright" data-test="copyrightNotice">&copy;  <span class="mc-variable okta-variables.Year variable">2022</span> <span class="mc-variable okta-variables.CompanyName variable">Okta, Inc</span>. All Rights Reserved. それぞれの商標は、それぞれの商標所有者に帰属します。
```

Comment:
I see in some files the year updated to 2023, but in some files, like this one, it is not. Might be worth investigating why

Reply:
I grepped, there are 201 files with the old 2022 date.
We had sent this job back with this issue specifically in mind.